### ISSUE TITLE
Enable cross-catalogue validation

### ISSUE THREAD

**Issue by Zhaph** [BSData/middle-earth#5]
From middleearth-sbg created by [Zhaph](https://github.com/Zhaph) : Zhaph/middleearth-sbg#4  

Once BattleScribe 2.02 is released, we need to be able to limit army selections so that you can't have an army with (for example) Gandalf the Grey (The Fellowship) and Gandalf the White (Minas Tirith).

This is waiting for the cross-catalogue reference feature in 2.02 before we can implement however.

**Comment by Hukoseft** [comment]
While I'm not going to close this issue, the files I have all adhere to this. With the future update it will become more elegant.

**Comment by Zhaph** [comment]
Yep, I've seen that @Hukoseft as did our original files - the idea being that going forward we can have more of a hierarchy:

```
Middle-Earth_Strategy_Battle_Game.gst (wargear, abilities, Saruman, etc.)
├ Good.cat (Aragorn, Gandalf, Elrond, Thorin's Company, etc.)
│  ├ Dwarves.cat* (Dwarf Warriors, Ring of Durin, etc.)
│  │    ├ Kingdom_of_Khazad-dum.cat
│  │    └ Kingdom_of_Moria.cat
│  ├ Elves.cat*
│  │    ├ Rivendell.cat
│  │    └ Lotlorien.cat
│  ├ Humans.cat*
│  │    ├ Rohan.cat
│  │    └ [...].cat
│  ├ The_Fellowship.cat
│  ├ The_Misty_Mountains.cat
│  └ [...].cat
└ Evil.cat
   └ [...].cat

* The race catalogues might be a step too far?
```

I'm not sure if the race options are a step too far or not, I guess we'll be able to see once we get more sight of the next version of Battlescribe.
